src/main/java/com/prgms/voucher/voucherproject/VoucherApp.java

```diff
@@ -0,0 +1,60 @@
+package com.prgms.voucher.voucherproject;
+
+import com.prgms.voucher.voucherproject.domain.VoucherType;
+import com.prgms.voucher.voucherproject.io.Console;
+import com.prgms.voucher.voucherproject.repository.MemoryVoucherRepository;
+import com.prgms.voucher.voucherproject.repository.VoucherRepository;
+import com.prgms.voucher.voucherproject.service.VoucherService;
+
+public class VoucherApp implements Runnable{
+    private final Console console = new Console();
+    private final VoucherService voucherService;
+    private static VoucherRepository voucherRepository;
+
+    public VoucherApp(VoucherService voucherService) {
+        this.voucherService = voucherService;
+        this.voucherRepository = voucherRepository;
+    }
+
+
+    public static void main(String[] args) {
+        voucherRepository = new MemoryVoucherRepository();
+        new VoucherApp(new VoucherService(voucherRepository)).run();
+    }
+
+    @Override
+    public void run() {
+        while(true){
+            console.printMenu();
+            switch (console.inputCommand().toLowerCase()){
```

Comment:
띄어쓰기가 빠져있어서 코드 제안 드렸어요.
```suggestion
            switch (console.inputCommand().toLowerCase()) {
```